code/modules/mob_spawn/ghost_roles/spider_roles.dm

```diff
@@ -168,7 +182,7 @@
 	directive = "Ensure the survival of the spider species and overtake whatever structure you find yourself in."
 	cluster_type = /obj/structure/spider/eggcluster/midwife
 	potentialspawns = list(
-		/mob/living/basic/spider/giant/midwife, // We don't want the event to end instantly because of a 2 hp spiderling dying
+		/mob/living/basic/spider/growing/spiderling/midwife, // We don't want the event to end instantly because broodmothers got a bad spawn
```

Comment:
I don't like this change. The only thing spawning midwives as spiderlings does is allow the round to end almost instantly because they have 2 hp, as the old comment states.
There is no gameplay benefit to the gamemode being delayed by the growth time of the initial spiders, nor the added risk of it ending before it can even begin.

Reply:
> I don't like this change. The only thing spawning midwives as spiderlings does is allow the round to end almost instantly because they have 2 hp, as the old comment states. There is no gameplay benefit to the gamemode being delayed by the growth time of the initial spiders, nor the added risk of it ending before it can even begin.

The main reason I want this to be introduced is mainly out of the fact that often broodmothers spawn in precarious situations and they are left to make a nest in a place that will meet their end letting them vent crawl for a short period lets them choose a good place to put the nest

Reply:
OK that's a reasonable justificaiton. 
To be honest some day we should just spawn the broodmothers as camera mobs (like Blob) who can choose their spawnpoint but that day is not today.

Reply:
> OK that's a reasonable justificaiton. To be honest some day we should just spawn the broodmothers as camera mobs (like Blob) who can choose their spawnpoint but that day is not today.

Next expansion I'm gona introduce the spider queen and I just might make their spiderling type since its unique, nearly invincible.

Although the camera AI like blob might be better